src/riscv.c

```diff
@@ -184,6 +186,31 @@ IO_HANDLER_IMPL(byte, write_b, W)
 #undef R
 #undef W
 
+#if RV32_HAS(JIT) && RV32_HAS(T2C)
```

Comment:
We can centralize the dependency in `src/feature.h` to ensure T2C depends on JIT, eliminating the need to write `#if RV32_HAS(JIT) && RV32_HAS(T2C)`.